neptune-core/src/application/json_rpc/core/api/rpc.rs

```diff
@@ -56,4 +63,75 @@ pub trait RpcApi: Sync + Send {
         &self,
         request: TipAnnouncementsRequest,
     ) -> TipAnnouncementsResponse;
+
+    async fn get_block_digest(&self, selector: RpcBlockSelector) -> GetBlockDigestResponse {
+        self.get_block_digest_call(GetBlockDigestRequest { selector })
+            .await
+    }
+    async fn get_block_digest_call(&self, request: GetBlockDigestRequest)
+        -> GetBlockDigestResponse;
+
+    async fn get_block(&self, selector: RpcBlockSelector) -> GetBlockResponse {
+        self.get_block_call(GetBlockRequest { selector }).await
+    }
+    async fn get_block_call(&self, request: GetBlockRequest) -> GetBlockResponse;
+
+    async fn get_block_proof(&self, selector: RpcBlockSelector) -> GetBlockProofResponse {
+        self.get_block_proof_call(GetBlockProofRequest { selector })
+            .await
+    }
+    async fn get_block_proof_call(&self, request: GetBlockProofRequest) -> GetBlockProofResponse;
+
+    async fn get_block_kernel(&self, selector: RpcBlockSelector) -> GetBlockKernelResponse {
+        self.get_block_kernel_call(GetBlockKernelRequest { selector })
+            .await
+    }
+    async fn get_block_kernel_call(&self, request: GetBlockKernelRequest)
+        -> GetBlockKernelResponse;
+
+    async fn get_block_header(&self, selector: RpcBlockSelector) -> GetBlockHeaderResponse {
+        self.get_block_header_call(GetBlockHeaderRequest { selector })
+            .await
+    }
+    async fn get_block_header_call(&self, request: GetBlockHeaderRequest)
+        -> GetBlockHeaderResponse;
```

Comment:
Can you show the categories here by just adding a comment? Like so:
/* Archival */